header-footer-grid/Core/Builder/Header.php

```diff
@@ -130,9 +149,181 @@ public function customize_register( WP_Customize_Manager $wp_customize ) {
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'        => self::BACKGROUND_HEADING,
+				'group'     => $section_id,
+				'label'     => esc_html__( 'Background', 'neve' ),
+				'section'   => $section_id,
+				'tab'       => 'style',
+				'priority'  => 15,
+				'transport' => 'postMessage',
+				'type'      => 'Neve\Customizer\Controls\Heading',
+				'options'   => [
+					'accordion'        => true,
+					'controls_to_wrap' => 20,
+					'expanded'         => true,
+					'class'            => 'background-accordion',
+				],
+			]
+		);
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'                 => self::ADVANCED_STYLE,
+				'group'              => $section_id,
+				'label'              => esc_html__( 'Enable Individual Row Background', 'neve' ),
+				'section'            => $section_id,
+				'tab'                => 'style',
+				'type'               => 'neve_toggle_control',
+				'priority'           => 25,
+				'transport'          => 'refresh',
+				'sanitize_callback'  => 'neve_sanitize_checkbox',
+				'default'            => true,
+				'conditional_header' => true,
+			]
+		);
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'                    => self::BACKGROUND_SETTING,
+				'group'                 => $section_id,
+				'section'               => $section_id,
+				'tab'                   => 'style',
+				'label'                 => esc_html__( 'Global Background', 'neve' ),
+				'description'           => esc_html__( 'A background color or image that spans across all header rows.', 'neve' ),
+				'type'                  => 'neve_background_control',
+				'live_refresh_selector' => true,
+				'live_refresh_css_prop' => [
+					'cssVar' => [
+						'vars'     => 'backgroundControl',
+						'selector' => '.global-styled',
+					],
+				],
+				'options'               => [
+					'priority'        => 30,
+					'active_callback' => [ $this, 'background_options_active_callback' ],
+				],
+				'default'               => [
+					'type'       => 'color',
+					'colorValue' => $this->default_background,
+				],
+				'transport'             => 'postMessage',
+				'sanitize_callback'     => 'neve_sanitize_background',
+				'conditional_header'    => true,
+			]
+		);
+
+		$rows = [ 'top', 'main', 'bottom' ];
+		foreach ( $rows as $row ) {
+			SettingsManager::get_instance()->add(
+				[
+					'id'                => $row . '_shortcut',
+					'group'             => $section_id,
+					'section'           => $section_id,
+					'tab'               => 'style',
+					'transport'         => 'postMessage',
+					'sanitize_callback' => 'esc_attr',
+					'type'              => '\Neve\Customizer\Controls\Button',
+					'priority'          => 35,
+					'options'           => [
+						// translators: %s row name
+						'text_before'      => sprintf( __( 'Set Background for the %s Row', 'neve' ), ucfirst( $row ) ),
+						'text_after'       => '.',
+						'button_text'      => __( 'here', 'neve' ),
```

Comment:
Let's change this around to be more mindful in regards to the strings we're adding:

- remove this `text_before` and `text_after` altogether;
- use the `title` available in the array values from the array I mentioned above as `button_text';
- add a property for `icon_class` and set that to `edit`;